### ISSUE TITLE
[LC0014] Locked = true

### ISSUE THREAD

**Issue by fvet** [StefanMaron/BusinessCentral.LinterCop#153]
Any clue why I keep getting the LC0014 warning for this permissionSet?
Does the rule check on 'Locked=true' and 'Locked = true' (no matter the spacing).

![image](https://user-images.githubusercontent.com/25268332/155488914-c09d9b4d-7da1-4902-986e-a54581ba6dda.png)

**Comment by StefanMaron** [comment]
Can not reproduce 
![image](https://user-images.githubusercontent.com/24838311/155504478-a60fe120-b2b0-409b-b6cd-35f42a4351da.png)
![image](https://user-images.githubusercontent.com/24838311/155504595-513adc1e-d987-4f1c-9e64-9ff2af4ace93.png)
![image](https://user-images.githubusercontent.com/24838311/155504531-0b918da3-7754-4fc2-9101-2593f1b49a8a.png)

**Comment by fvet** [comment]
Included some sample code to repro.

![image](https://user-images.githubusercontent.com/25268332/155893441-4f8fcf38-e466-4047-b5e8-b0f918c4f7f5.png)

```
permissionSet 71096875 "ESCD PLAN DISPATCH"
{
    Access = Internal;
    Assignable = true;
    Caption = 'Dynavision Planning Dispatch', Locked = true;

    Permissions =
         Table "Customer" = X;
}
```

Decreased caption length, added MaxLength = 30, but still get the warning.

![image](https://user-images.githubusercontent.com/25268332/155893502-29e15ab2-82fe-4a61-a2ca-d78d82ce0cc2.png)

**Comment by StefanMaron** [comment]
is this also related to AL Runtime version 9 ?

**Comment by fvet** [comment]
This issue was logged while running on AL runtime v9.
On v8.4.x it does not occur.

**Comment by StefanMaron** [comment]
This should be fixed now with version [v0.25.1](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.25.1)
